truffle/contracts/Splitter.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity 0.5.10;
+
+import './Owned.sol';
+
+contract Splitter is Owned {
+    
+    struct Person {
+        address wallet;
```

Comment:
This key field would be redundant if the mapping at line 15 is also the address.